src/components/data-table/column-sorting/column-sort.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Define column sort types/models
+ */
+
+/**
+ * Specifies sorting direction
+ */
+export enum SortDirection {
+  ASCEND, DESCEND
+}
+
+/**
+ * Sorting model.
+ */
+export interface IColumnSortingModel {
```

Comment:
Interface shouldn't be prefixed by _I_ ([styleguide](https://angular.io/styleguide#!#interfaces))